admin_manual/configuration_database/db_conversion.rst

```diff
@@ -11,23 +11,17 @@ testing and simple single-user ownCloud servers, but it does not scale for multi
 Run the conversion
 ------------------
 
-First setup the new database, here called "new_db_name".
-In ownCloud root folder call
+After you have setup the new database, in the ownCloud root folder run the following command to convert the database to the new format:
 
 ::
 
   php occ db:convert-type [options] type username hostname database
 
-The Options
 
-* ``--port="3306"``                       the database port (optional)
-* ``--password="mysql_user_password"``    password for the new database. If omitted the tool will ask you (optional)
-* ``--clear-schema``                      clear schema (optional)
-* ``--all-apps``                          by default, tables for enabled apps are converted, use to convert also tables of deactivated apps (optional)
```

Comment:
Did you intend to remove the list of possible options?
(Maybe they are already somewhere else?)